src/tests/functional/shared_test_classes/src/single_layer/generate_proposals.cpp

```diff
@@ -16,8 +17,9 @@ std::ostream& operator <<(
         const ov::op::v9::GenerateProposals::Attributes& attributes) {
     ss << "score_threshold=" << attributes.min_size << "_";
     ss << "nms_threshold=" << attributes.nms_threshold << "_";
-    ss << "max_delta_log_wh=" << attributes.post_nms_count << "_";
-    ss << "num_classes=" << attributes.pre_nms_count;
+    ss << "post_nms_count=" << attributes.post_nms_count << "_";
+    ss << "pre_nms_count=" << attributes.pre_nms_count;
+    ss << "normalized=" << attributes.normalized;
```

Comment:
"roi_num_type" is also required.
And I think we can add "nms_eta" for the single layer test.

Reply:
Added nms_eta here. It doesn't make much sense to add it as a parameter for tests since nms_eta is forced to always be 1.0 - see https://github.com/openvinotoolkit/openvino/blob/master/src/core/src/op/generate_proposals.cpp#L52

Reply:
Got it . Understood.